python/GafferUI/VectorDataWidget.py

```diff
@@ -57,7 +57,13 @@ class VectorDataWidget( GafferUI.Widget ) :
 	# of identical length.
 	#
 	# header may be False for no header, True for a default header, or a list of
-	# strings to specify a custom header per column.
+	# strings to specify a custom header per column. If the VectorDataWidget data is set to
```

Comment:
Just in case I'm not making myself clear, I'm suggesting that the API should expect `len( data ) == len( header ) == len( columnToolTips )`, and that it is poor documentation that suggested they were otherwise.